spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/sql/CloudSqlEnvironmentPostProcessor.java

```diff
@@ -67,8 +69,20 @@ public void postProcessEnvironment(ConfigurableEnvironment environment, SpringAp
 				LOGGER.info("post-processing Cloud SQL properties for + " + databaseType.name());
 			}
 
-			// Bind properties without resolving placeholders
-			Binder binder = new Binder(ConfigurationPropertySources.get(environment), null, null, null, null);
+			// Bind properties without resolving Secret Manager placeholders
+			Binder binder = new Binder(ConfigurationPropertySources.get(environment), new PlaceholdersResolver() {
```

Comment:
Want to refactor this into a nested class? It's not so trivial, plus there are additional parameters after the anonymous class...